src/bridge/views/SentTransaction.js

```diff
@@ -146,7 +143,7 @@ const SentTransaction = props => {
           prop-type={'link'}
           onClick={() => {
             setPointCursor(pointCursor);
-            popRoute();
+            history.pop(2);
```

Comment:
@Fang- what's the correct behavior for when a user executes a stateless transaction, finds themselves at SentTransaction, and then presses "ok"? right now, with a single pop, I go back to the statelesstransaction component instead of the point view.

the behavior in the current release goes back to the point view, but the implementation of popRoute was only popping a single entry: https://github.com/urbit/bridge/blob/master/src/bridge/Bridge.js#L179

so there's a bug (?) here where statelesstransaction is in the history when it shouldn't be, I think

Reply:
figured it out — the functions we're passing to these consumers close-over the current value of `history` so when we have code like:

```
history.pop()
history.push(...)
```

the second function call's result is persisted in state, but since it closes over the state during the previous render, we end up ignoring the `.pop` call. The fix is to use the reducer-esq functionality of setState, which is coming in a next commmit

Reply:
Correct behavior is definitely to kick you back to the point view. So popping twice seems right.